website/docs/commands/fmt.md

```diff
@@ -20,29 +20,26 @@ You can check formatting correctness using a `--check` flag:
 scala-cli fmt --check
 ```
 
-### Dialects
+### Scalafmt version and dialect
 
-Scala CLI also supports dialects that are passed to the formatter.
-This value is only used if there is no `.scalafmt.conf` file.
-However, if it exists, then all configuration should be placed there.
-For a list of all possible values, consult
-the [official Scala Dialects documentation](https://scalameta.org/scalafmt/docs/configuration.html#scala-dialects):
+Configuration file with specified **version** and **dialect** is required to run `scalafmt`. Scala CLI `fmt` tries to find the `.scalafmt.conf` file in the **projects root** directory and **current workspace** directory. There are 3 possible cases:
 
-```bash
-scala-cli fmt --dialect scala212
-```
-
-### Scalafmt version
+1. Configuration file with the specified version and dialect is found.
+2. Configuration file is found, but it doesn't have specified version or dialect.
```

Comment:
NIT: technically, you missed the few extra cases when one is found, but not the other

Reply:
Isn't subsection 2 covering it?

Reply:
it is, just pointing out there's actually more than 3 cases, which create a slight dissonance here